src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx

```diff
@@ -0,0 +1,209 @@
+import { ActionType, ContextModule, OwnerType } from "@artsy/cohesion"
+import { Button, Spacer, Text, useToasts } from "@artsy/palette"
+import { COUNTRY_CODES } from "Utils/countries"
+import { Form, Formik } from "formik"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { useSystemContext } from "System"
+import { useRouter } from "System/Router/useRouter"
+import createLogger from "Utils/logger"
+import { recaptcha, RecaptchaAction } from "Utils/recaptcha"
+import {
+  ConsignmentInquiryForm,
+  ConsignmentInquiryFormModel,
+} from "Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryForm"
+import { SentConsignmentInquiry } from "@artsy/cohesion/dist/Schema/Events/Consignments"
+import { ConsignmentInquiry_me$data } from "__generated__/ConsignmentInquiry_me.graphql"
+import { createConsignmentInquiry } from "Apps/Consign/Routes/ConsignmentInquiry/utils/createConsignmentInquiry"
+import {
+  consignmentInquiryValidationSchema,
+  validate,
+} from "Apps/Consign/Routes/ConsignmentInquiry/utils/validation"
+import { CreateConsignmentInquiryMutationInput } from "__generated__/createConsignmentInquiryMutation.graphql"
+import { ConsignmentInquiryFormAbandonEditModal } from "Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryFormAbandonEdit"
+import { useState } from "react"
+import { TopContextBar } from "Components/TopContextBar"
+
+const logger = createLogger("ConsignmentInquiry/ConsignmentInquiry.tsx")
+
+const getContactInformationFormInitialValues = (
+  me: ConsignmentInquiry_me$data
+): ConsignmentInquiryFormModel => ({
+  name: me?.name || "",
+  email: me?.email || "",
+  phoneNumber:
+    me?.phoneNumber?.national ||
+    me?.phone ||
+    me?.phoneNumber?.international ||
+    "",
+  phoneNumberCountryCode: me?.phoneNumber?.regionCode || "us",
+  message: "",
+})
+
+export interface ConsignmentInquiryProps {
+  me: ConsignmentInquiry_me$data
+}
+
+export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
+  me,
+}) => {
+  const [showModal, setShowModal] = useState(false)
```

Comment:
**suggestion:** It might be easier to read if we specify which modal will be shown in the name.

```suggestion
  const [showExitModal, setShowExitModal] = useState(false)
```